src/react_native/reanimated.cljs

```diff
@@ -90,8 +79,10 @@
 ;; so first convert kebab case styles into camel case styles
 (defn apply-animations-to-style
   [animations style]
-  (let [animations (map-keys kebab-case->camelCase animations)
-        style      (apply dissoc (map-keys kebab-case->camelCase style) (keys animations))]
+  (let [animations (utils.collection/map-keys csk/->camelCaseString animations)
```

Comment:
could you please elaborate on this change? cc @Parveshdhull

Reply:
I think we are migrating to library function instead of using ours. This should ok for now.
But while fixing [issue](https://github.com/status-im/status-mobile/issues/14756) (nil value in reanimated style crashing at runtime), I am also cleaning `apply-animations-to-style` function, so we don't need to convert this anymore, this will be handled by worklet itself. [code](https://github.com/status-im/status-mobile/commit/248b41022e11c80a1cac0e5d8a6e6db828b0127f)